pkgs/applications/misc/plasma-theme-switcher/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  stdenv, lib, cmake, extra-cmake-modules, fetchFromGitHub, qtbase, kdeFrameworks
+}:
+
+stdenv.mkDerivation rec {
+  pname = "plasma-theme-switcher";
+  version = "0.1";
+  dontWrapQtApps = true;
```

Comment:
This is only used with `wrapQtAppsHook`, i think it can be removed
```suggestion
```

Reply:
It refuses to compile without that line.

`Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.` in the qtPreHook

Reply:
Huh, i wasn't aware that's a thing. Thanks for letting me know :)